assets/sass/components/_forms.scss

```diff
@@ -133,6 +133,15 @@
     display: block;
     margin-bottom: 5px;
     font-weight: bold;
+
+    &.ff-label--append:after {
+        margin-left: 5px;
+        content: attr(data-append);
```

Comment:
Oh! Why didn't I think of doing this before. I added a `ff-label-note` class a while back that had the text inline in the html when I translated the forms, but this is nicer. Might circle back and change my original stuff to match this.